index.html

```diff
@@ -0,0 +1,140 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+
+  <link rel="stylesheet" href="css/global.css">
+  <link rel="stylesheet" href="./css/navsection.css">
+  <link rel="stylesheet" href="./css/homesection.css">
+  <link rel="stylesheet" href="./css/programsection.css">
+  <link rel="stylesheet" href="./css/speakersection.css">
+  <link rel="stylesheet" href="./css/aboutsection.css">
+
+
+  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@4.0.0/dist/css/bootstrap.min.css" integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm" crossorigin="anonymous">
+  <script src="https://kit.fontawesome.com/bdd8845187.js" crossorigin="anonymous"></script>
+  <title>Masters of Philosophy</title>
+</head>
+
+<body>
+<main>
+<header>
+  <nav class="navigation d-flex align-items-center justify-content-between">
+    <i id="open-menu" class="fa-solid fa-bars open-menu"></i>
+    <a href="">MoP.™</a>
+    <ul class="navigation__list">
+      <li><a href="./about.html">About</a></li>
+      <li><a href="#program">Program</a></li>
+      <li><a href="#speakers">Speakers</a></li>
+    </ul>
+  </nav>
+</header>
+<ul class="navigation__mobile">
+  <li><a href=""><img id="close-menu" class="close-menu" src="./images/close.png" alt="close"></a></li>
+   <li><a href="./about.html" class="navigation__mobile__links externalLink">About</a></li>
+   <li><a onclick="openLinks('#program');" class="navigation__mobile__links internalLinks">Program</a></li>
+   <li><a onclick="openLinks('#speakers');" class="navigation__mobile__links internalLinks">Speakers</a></li>
+</ul>
+<section class="home p-4">
+  <p class="home__para1">
+    "Greetings, students of Philosophy"
+  </p>
+  <h1>
+    Masters of Philosophy <br>
+    ANNUAL CONGRESS OF DISCOURSE <br>
+    2022 <br>
+  </h1>
+  <p class="home__para2 p-3">
+    An event for all who enjoy the timeless lectures 
+    and scholarly wisdom of Ancient Philosophers brought to life 
+    through the brilliance of hologram technology. 
+  </p>
+  <h2>
+    2020.06.19 (MON) ~ 20(TUES)
+  </h2>
+  <h3>
+    @ The Seneca Hall of Congress, Athens
+  </h3>
+</section>
+
+<section id="program" class="program p-4">
+  <h2 class="text-center p-3 ml-auto mr-auto">
+    Main Program
+  </h2>
+  <div class="program__container m-3">
```

Comment:
As your previews reviewer mentioned, "_Please add a background image to this section. It doesn't have to be exactly like the Behance design 🙏_", kindly add a background image to this section so that your project design its almost exactly as the guideline 🚀